helm/reana/templates/reana-server.yaml

```diff
@@ -54,9 +54,10 @@ spec:
           - mountPath: {{ .Values.shared_storage.shared_volume_mount_path }}
             name: reana-shared-volume
           {{- range $workspace_paths := .Values.workspaces.paths }}
-          {{- if not (eq $workspace_paths "/var/reana")}}
-          - name: {{ $workspace_paths | quote |replace "/" "" }}
-            mountPath: {{ $workspace_paths }}
+          {{- $workspace_path := split ":" $workspace_paths }}
+          {{- if and (not (eq $workspace_path._0 "/var/reana")) (not (eq $workspace_path._1 "/var/reana"))}}
```

Comment:
Pehaps `{{- if ne $workspace_path._1 "/var/reana" }}` would be enough here? Ideally we could compare with `.Values.shared_storage.shared_volume_mount_path` not to hardcode the values

Reply:
I think the comparison with both `$workspace_path._0` and `$workspace_path._1` is needed, as it would be a problem to mount `/var/reana` either as a `mountPath` or `hostPath`